src/plugins/GUITestBase/src/GTUtilsProjectTreeView.cpp

```diff
@@ -192,14 +192,13 @@ void GTUtilsProjectTreeView::scrollToIndexAndMakeExpanded(HI::GUITestOpStatus &o
 
     class MainThreadActionExpand : public CustomScenario {
     public:
-        MainThreadActionExpand(QTreeView *treeView, const QModelIndex &index)
-            : CustomScenario(), treeView(treeView), index(index) {
+        MainThreadActionExpand(QTreeView *_treeView, const QModelIndex &_index)
+            : treeView(_treeView), index(_index) {
```

Comment:
Not sure it's good - you've removed the initialization of the base class. The all base class members are invalid now.

Reply:
I think we already discussed it some PRs ago.  Please read the standard: https://isocpp.org/std/the-standard.